### DISCUSSION TITLE
503 errors on my pod deployed to k8s cluster. All files are statically generated.

### DISCUSSION THREAD

**Discussion by sanalpanicker** [vercel/next.js#52106] [App Router]
Hi All,

I am intermittently getting (especially on route changes and page reloads) 503. I also sometimes get chunk load error chunks/webpack.js.
I have a middleware.ts doing authentication calls. Is this causing some timeouts?

As suggested in different blogs - it's a large-scale production deployment, and I cannot always delete .next cache and ask users to do a hard refresh every time.

**Comment by xveganxxxedgex** [comment]
I'm having similar issues after upgrading to next v13 - running the dev environment seems to work fine, but when I run `next start`, I'm getting a lot of intermittent 503 issues. It seems to happen when doing a mix of normal and hard refreshes. Sometimes the 503s don't cause issues, but sometimes on hard refresh it will 503 immediately and cause the page to get stuck just displaying a white screen. In case it matters, this app is still only using the `pages` dir.

![Screenshot 2023-07-12 at 3 21 39 PM](https://github.com/vercel/next.js/assets/4555168/f9cbf390-0782-4061-b441-623208560a2c)

![Screenshot 2023-07-12 at 3 21 52 PM](https://github.com/vercel/next.js/assets/4555168/b2bdb682-d234-4be0-b817-6b9ca4394be0)
